src/mad_spark_alt/core/answer_extractor.py

```
@@ -0,0 +1,405 @@
+        ],
+        'why': [
+            r'^why',
+            r'reason',
+            r'cause',
+        ],
+        'best_practice': [
+            r'best\s+(way|method|practice|approach)',
+            r'most\s+effective',
+            r'optimal',
+        ]
+    }
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)